models/packages/package.go

```diff
@@ -175,6 +175,21 @@ func (pt Type) SVGName() string {
 	panic(fmt.Sprintf("unknown package type: %s", string(pt)))
 }
 
+// String converts the package type to a string
+func (pt Type) String() string {
+	return string(pt)
+}
+
+// GetPackageTypeByString returns the type for the given string
+func GetPackageTypeByString(typeName string) *Type {
+	for _, currentType := range TypeList {
+		if currentType.String() == typeName {
+			return &currentType
+		}
+	}
+	return nil
+}
+
```

Comment:
It is possible to reduce the number of lines  a bit.

```suggestion

// GetPackageTypeByString returns the type for the given string
func GetPackageTypeByString(typeName string) *Type {
	for _, currentType := range TypeList {
		if string(currentType) == typeName {
			return &currentType
		}
	}
	return nil
}
```